website/docs/faq.md

```diff
@@ -117,7 +117,8 @@ Upgrading Prysm is done differently depending on your operating system and insta
 
 #### How can I downgrade Prysm to an older version?
 
-Upgrading Prysm is done differently depending on your operating system and installation method. Please note that downgrading may not be as easy as upgrading as some versions may not be backward compatible and you will need to perform extra steps. For example, migrating down from v1.x to v1.0.x has breaking changes that require you to also rollback your database.
```

Comment:
Display is strange:

![image](https://github.com/prysmaticlabs/documentation/assets/4943830/ada6d7ae-2daf-46c0-82a7-b69e2616c2fd)

Reply:
good catch